Misc/NEWS.d/next/Library/2019-11-11-21-43-06.bpo-27805.D3zl1_.rst

```diff
@@ -0,0 +1,2 @@
+Allow opening pipes and other non-seekable files in append mode with
+:func:`open`.
```

Comment:
maybe explicitly mention that ESPIPE is ignored?

Reply:
I'm trying to describe the user-visible behavior change not the implementation.